userProfile/privateProfile.mjs

```diff
@@ -29,29 +30,26 @@ router.use(
   })
 )
 
-router.get("/profile", async (req, res) => {
-  const user = await req.user 
-  if (!user) return respondWithError(res, 401, "Unauthorized user") 
+router.get("/profile", auth0Middleware(), async (req, res) => {
+  const user = await req.user
+  if (!user) return respondWithError(res, 401, "Unauthorized user")
   const userObj = new User(user._id)
   const userProfile = await userObj.getSelf()
-  res.set("Content-Type", "application/json; charset=utf-8") 
+  res.set("Content-Type", "application/json; charset=utf-8")
 
   res.status(200).json(userProfile)
 })
 
-router.get("/projects", async (req, res) => {
-  const {_id} = await req.user 
-
-  if (!_id) return respondWithError(res, 401, "Unauthorized user")
+router.get("/projects", auth0Middleware(), async (req, res) => {
+  const user = await req.user
+  if (!user) return respondWithError(res, 401, "Unauthorized user") 
 
-  const userObj = new User(_id) 
+  const userObj = new User(user._id)
   const userProjects = await userObj.getProjects()
```

Comment:
You should add a mechanism to these in case userObj.getProjects() fails so that this endpoint can respond RESTfully about what error may have occurred.  Without any handling on this, it could cause an incorrect response or even a code crash.